### PR TITLE
Minor fixes for release

### PR DESCRIPTION
**Pull request by inodb**
- remove PDF button (fix cbioportal/cbioportal#3258)
- return border to what it was (fix cbioportal/cbioportal#3257)
- fix Lazy MobX table test

### CODE REVIEW

File: src/shared/components/lollipopMutationPlot/LollipopMutationPlot.tsx [hunk 1]
```diff
@@ -414,13 +414,12 @@ export default class LollipopMutationPlot extends React.Component<ILollipopMutat
                             <button className="btn btn-default btn-xs" onClick={this.handlers.handleSVGClick}>
                                 SVG <i className="fa fa-cloud-download" aria-hidden="true"></i>
                             </button>
-
-
+                            /* TODO: output real PDF instead of embedded picture PDF
```
Comment (onursumer) [inline src/shared/components/lollipopMutationPlot/LollipopMutationPlot.tsx]: This PDF on mutation tab is real. I think we want to disable the one on the Cancer Summary tab for now.

Comment (inodb) [reply src/shared/components/lollipopMutationPlot/LollipopMutationPlot.tsx]: Thanks @onursumer wrong one

File: src/shared/components/cancerSummary/CancerSummaryContent.tsx [hunk 2]
```diff
@@ -466,10 +466,12 @@ export class CancerSummaryContent extends React.Component<ICancerSummaryContentP
                         href={this.pngAnchor} download="cBioPortalCancerSummary.png">
                         PNG <i className="fa fa-cloud-download" aria-hidden="true"></i>
                     </a>
+                    /*
```
Comment (adamabeshouse) [inline src/shared/components/cancerSummary/CancerSummaryContent.tsx]: have you confirmed this works? I'm pretty sure this is invalid JSX and you need to wrap in enclosing curly brackets

File: src/shared/components/lazyMobXTable/LazyMobXTable.spec.tsx [hunk 3]
```diff
@@ -1240,26 +1240,26 @@ describe('LazyMobXTable', ()=>{
         it("shows the right text before the paging buttons", ()=>{
             let table = mount(<Table columns={simpleColumns} data={[]}/>);
             assert.equal(getItemsPerPage(table), 50, "confirm 50 items per page");
-            assert.equal(getTextBeforeButtons(table), "0-0 of 0");
+            assert.equal(getTextBeforeButtons(table), "Showing 0-0 of 0");
```
Comment (adamabeshouse) [inline src/shared/components/lazyMobXTable/LazyMobXTable.spec.tsx]: ahhh, yes

File: src/shared/components/cancerSummary/CancerSummaryContent.tsx (hunk 2 above)
Comment (inodb) [reply src/shared/components/cancerSummary/CancerSummaryContent.tsx]: @adamabeshouse you are right, good tip about the curly braces. The errors you can make in three lines of code :)

**Pull request merged**